scripts/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+if [ -z "$1" ]; then 
+    BUILD_TYPE=Release
+else 
+    BUILD_TYPE="$1"
+fi
+
+PWD=$(pwd)
```

Comment:
Please do not assume a specific working directory, instead use `$(dirname $0)` to determine the location of the source files.